website/docs/plugin/testing/acceptance-tests/plan-checks/output.mdx

```diff
@@ -90,13 +110,111 @@ func TestExpectKnownOutputValue_CheckPlan_Bool(t *testing.T) {
 }
 ```
 
-## Example using `plancheck.ExpectUnknownOutputValue`
+## `ExpectNullOutputValue` Plan Check
+
+The built-in [`plancheck.ExpectNullOutputValue(address)`](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-testing/plancheck#ExpectNullOutputValue) plan check determines whether an output at the specified address has a null value.
+
+```go
+package example_test
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-plugin-testing/plancheck"
+)
+
+func Test_ExpectNullOutputValue_StringAttribute_NullConfig(t *testing.T) {
+	t.Parallel()
+
+	resource.Test(t, resource.TestCase{
+		// Provider definition omitted.
+		Steps: []resource.TestStep{
+			{
+				Config: `resource "test_resource" "test" {
+					string_attribute = null
+				}
+
+				output "string_attribute" {
+					value = test_resource.test.string_attribute
+				}
+				`,
+				ConfigPlanChecks: resource.ConfigPlanChecks{
+					PreApply: []plancheck.PlanCheck{
+						plancheck.ExpectNullOutputValue("string_attribute"),
+					},
+				},
+			},
+		},
+	})
+}
+```
+
+## `ExpectNullOutputValueAtPath` Plan Check
```

Comment:
Do we want to remove this documentation since we've recently decided to deprecate this? Or at least a note suggesting what to use instead.

Reply:
Have added a note for now.